tests/e2e/catalog/items/test_async_item.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+from mpt_api_client.exceptions import MPTAPIError
+from mpt_api_client.rql.query_builder import RQLQuery
+
+pytestmark = [pytest.mark.flaky]
+
+
+@pytest.fixture
+async def async_created_item(logger, async_mpt_vendor, item_data):
+    service = async_mpt_vendor.catalog.items
+    item = await service.create(item_data)
+    yield item
+    try:
+        await service.delete(item.id)
+    except MPTAPIError as error:
+        print(f"TEARDOWN - Unable to delete item {item.id}: {error.title}")  # noqa: WPS421
+
+
+def test_create_item(async_created_item):
+    assert async_created_item.name == "e2e - please delete"
+
+
+async def test_update_item(async_mpt_vendor, async_created_item):
+    service = async_mpt_vendor.catalog.items
+    update_data = {"name": "e2e - delete me (updated)"}
+    item = await service.update(async_created_item.id, update_data)
+    assert item.name == "e2e - delete me (updated)"
```

Comment:
Would it be worth as an extra check getting the item by name to make sure it's been really updated?